src/GusApi/Client/SoapClient.php

```diff
@@ -4,9 +4,12 @@
 
 class SoapClient extends \SoapClient
 {
-    public function __doRequest($request, $location, $action, $version, $one_way = 0): string
+    public function __doRequest($request, $location, $action, $version, $oneWay = false): string
```

Comment:
I think it was historically named `$one_way` and the type should be `int`. It was changed in PHP 8 but we should support PHP 7.1+ https://www.php.net/manual/en/soapclient.dorequest.php#refsect1-soapclient.dorequest-changelog